packages/abi-typegen/test/fixtures/forc-projects/full/src/main.sw

```diff
@@ -1,5 +1,6 @@
 contract;
-
+use libs_for_testing::ExternalStruct;
+use libs_for_testing::ExternalEnum;
```

Comment:
These types are used for non-std cases. For example, this type produces an abi type with the name `libs-for-testing::ExternalEnum`.